src/udp_wrap.cc

```diff
@@ -284,12 +296,16 @@ void UDPWrap::DoSend(const FunctionCallbackInfo<Value>& args, int family) {
   if (err == 0) {
     err = uv_udp_send(&req_wrap->req_,
```

Comment:
Not on this PR, but it would probably be nice to use [uv_udp_try_send](http://docs.libuv.org/en/v1.x/udp.html#c.uv_udp_try_send) here, just in case we can send it on the spot.